src/unit_tests/wazuh_db/test_wdb.c

```diff
@@ -694,47 +705,95 @@ void test_wdb_init_stmt_in_cache_invalid_statement(void **state) {
     assert_null(result);
 }
 
-int main()
-{
-    const struct CMUnitTest tests[] =
-    {
-        //wdb_open_tasks
+void test_wdb_get_internal_config() {
+    cJSON *ret = wdb_get_internal_config();
+    assert_true(cJSON_IsObject(ret));
+
+    cJSON* root = cJSON_GetObjectItem(ret, "wazuh_db");
+    assert_true(cJSON_IsObject(root));
+
+    cJSON *c1 = cJSON_GetObjectItem(root, "commit_time_max");
+    assert_true(cJSON_IsNumber(c1));
+    cJSON *c2 = cJSON_GetObjectItem(root, "commit_time_min");
+    assert_true(cJSON_IsNumber(c2));
+    cJSON *c3 = cJSON_GetObjectItem(root, "open_db_limit");
+    assert_true(cJSON_IsNumber(c3));
+    cJSON *c4 = cJSON_GetObjectItem(root, "sock_queue_size");
+    assert_true(cJSON_IsNumber(c4));
+    cJSON *c5 = cJSON_GetObjectItem(root, "worker_pool_size");
+    assert_true(cJSON_IsNumber(c5));
+
+    cJSON_Delete(ret);
+}
+
+void test_wdb_get_config(){
+    cJSON *ret = wdb_get_config();
+
+    cJSON *root = cJSON_GetObjectItem(ret, "wdb");
+    assert_true(cJSON_IsObject(root));
+
+    cJSON *cfg_array = cJSON_GetObjectItem(root, "backup_settings_nodes");
+    assert_true(cJSON_IsArray(cfg_array));
+
+    int size = cJSON_GetArraySize(cfg_array);
+    for (int i = 0; i < size; ++i) {
```

Comment:
By using `cJSON_ArrayForEach` we can avoid getting the size of the array, and then the array item. In this way, we clean the code a bit.

Reply:
done!